types/event.go

```diff
@@ -241,8 +241,14 @@ func (sp *Span) GetDataSize() int {
 
 	if sp.IsDecisionSpan() {
 		if v, ok := sp.Data["meta.refinery.span_data_size"]; ok {
-			return v.(int)
+			switch value := v.(type) {
```

Comment:
I'm not sure this is related to the new trace locality option. Is this a bug that's got caught up in this PR?